test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 nose
 mock
 flask<0.12
+django<1.11
```

Comment:
Hmm. I think specifying a lower bound to be specific about what Django versions the contrib module "supports" might be useful.